plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -302,3 +307,49 @@ func TestBackoff(t *testing.T) {
 		t.Errorf("expected: 1, got %s", duration.String())
 	}
 }
+
+// TestResponsibleForPod tests if a pod with an annotation that should cause it to
+// be picked up by the default scheduler, is in fact picked by the default scheduler
+func TestResponsibleForPod(t *testing.T) {
```

Comment:
Can we create multiple factories and then use the test to show that 
- if no annotation is specified, then default will be chosen
- pick scheduler based on name
- show what happens for the wrong scheduler name